### DISCUSSION TITLE
滚动条问题

### DISCUSSION THREAD

**Discussion by ksmarsh** [rennzhang/codemirror-editor-vue3#27] [General]
我在使用这个组件的时候 ，滑动滚动条总是不协调 并且会在某个位置一直抖动 相同的代码 放在一个demo的页面却没有这个问题 ，请教一下 是什么原因导致的，有解决的办法吗？

**Comment by rennzhang** [comment]
试试修改高度、字体大小、行高

**Reply by ksmarsh** [reply]
![image](https://user-images.githubusercontent.com/107527519/184817811-bfd882f5-3b99-435b-80bb-98adbc118385.png)
这是我写的例子 放在两个项目里一个正常一个滚动条出现问题，尝试过修改高度 ，字体大小 并不行，没有经过封装的codemirror5也有这个问题

**Reply by ksmarsh** [reply]
你好，我想问下 怎么使用函数操作 上一个差异goPrevDiff，下一个差异goNextDiff 命令 看了Codemirror里的配置，并没有这两个命令
